src/base/utils/net.h

```diff
@@ -29,10 +29,10 @@
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
 
+#include <QHostAddress>
```

Comment:
Why did you change it?

Reply:
Isn't that the usual behavior when using the forward declaration?
We forward declare something and then we include an appropriate header in place where it is really used (i.e. in "preferences.cpp" in this case).

Reply:
Okay, then.
But it really confused me why we use this "impersonal" type (QPair) as Subnet. It's good for using in template algorithms, but here I'd rather see some structure with meaningful field names.